backup-tool/backup-command/src/main/java/org/apache/iotdb/backup/command/ExportTool13_1.java

```diff
@@ -0,0 +1,340 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */ package org.apache.iotdb.backup.command;
+
+import org.apache.iotdb.backup.command.Exception.ArgsErrorException;
+import org.apache.iotdb.backup.command.utils.AbstractCsvTool;
+import org.apache.iotdb.backup.core.ExportStarter;
+import org.apache.iotdb.backup.core.exception.ParamCheckException;
+import org.apache.iotdb.backup.core.pipeline.context.model.CompressEnum;
+import org.apache.iotdb.backup.core.pipeline.context.model.ExportModel;
+import org.apache.iotdb.backup.core.pipeline.context.model.FileSinkStrategyEnum;
+import org.apache.iotdb.rpc.IoTDBConnectionException;
+import org.apache.iotdb.rpc.StatementExecutionException;
+
+import org.apache.commons.cli.*;
+import reactor.core.Disposable;
+
+import java.io.File;
+
+/** @Author: LL @Description: @Date: create in 2022/7/7 14:29 */
+public class ExportTool13_1 extends AbstractCsvTool {
```

Comment:
what is 13_1, 0.13.1?
No need to show these in class name, the minor version do not change the TsFile and IoTDB structure/interface.
So only put these code in rel/0.13 is enough.

Reply:
already modified